pkg/porter/explain.go

```diff
@@ -22,14 +23,15 @@ type ExplainOpts struct {
 
 // PrintableBundle holds a subset of pertinent values to be explained from a bundle.Bundle
 type PrintableBundle struct {
-	Name         string                `json:"name" yaml:"name"`
-	Description  string                `json:"description,omitempty" yaml:"description,omitempty"`
-	Version      string                `json:"version" yaml:"version"`
-	Parameters   []PrintableParameter  `json:"parameters,omitempty" yaml:"parameters,omitempty"`
-	Credentials  []PrintableCredential `json:"credentials,omitempty" yaml:"credentials,omitempty"`
-	Outputs      []PrintableOutput     `json:"outputs,omitempty" yaml:"outputs,omitempty"`
-	Actions      []PrintableAction     `json:"customActions,omitempty" yaml:"customActions,omitempty"`
-	Dependencies []PrintableDependency `json:"dependencies,omitempty" yaml:"dependencies,omitempty"`
+	Name          string                `json:"name" yaml:"name"`
+	Description   string                `json:"description,omitempty" yaml:"description,omitempty"`
+	Version       string                `json:"version" yaml:"version"`
+	PorterVersion string                `json:"porterVersion" yaml:"porterVersion"`
```

Comment:
Let's make it so that if this isn't a Porter bundle, then `porterVersion` is not included when we marshal it to json or yaml. We can do that by using omitempty

```
PorterVersion string                `json:"porterVersion,omitempty" yaml:"porterVersion,omitempty"`
```